docs/src/index.md

```diff
@@ -525,11 +525,165 @@ To reference columns with more complicated expressions, you must wrap column ref
 @transform df :a + $(get_column_name(x))
 ```
 
-## Constructing multi-column arguments and `src => fun => dest` calls using `$`
+## Operations with multiple columns at once using `AsTable` inside operations
```

Comment:
Again, why not put this section right after the one about `@astable`?

Reply:
Thanks. Re-arranged.